user/mods/ServerValueModifier/Presets/The Escape.json

```diff
@@ -288,28 +288,28 @@
     "RaidTime": 0,
     "SaveQuestItems": false,
     "Exfils": {

```

Comment:
all exfils change should not be touched (we have already openextracts and ptt for this)